### ISSUE TITLE
Improve followed - unfollowed status

### ISSUE THREAD

**Issue by dtonon** [mikedilger/gossip#795]
I propose to review, and hopefully improve, the followed - unfollowed status.

Currently we display a little circle with the accent color at top-right:

<img width="439" alt="image" src="https://github.com/mikedilger/gossip/assets/89577423/7f8c5f6f-a031-4137-a859-52ecbbf31494">
<br/><br/>

This format has two problems:

1) Statically most profiles are followed and so have that sign, this make less evident if it is missing
2) The dot in that position remembers the online status of some chats, and this could be confusing

The solution:

1) Flag only non followed users
2) Use a triangle shape for the flag, outside the picture (so it cannot be forged), like that:

<img width="961" alt="image" src="https://github.com/mikedilger/gossip/assets/89577423/8683ebb5-6363-4742-a034-032901fad119">
<br/><br/>

In this example we can clearly spot a impersonator of fiatjaf, and two random unknown users.

Maybe the flag color may vary following the WoT, if if one day it is implemented, so it could be easy to recognize trusted users from uncertain ones.

@mikedilger @bu5hm4nn opinions?

**Comment by mikedilger** [comment]
I agree with your assessment of a problem.  But maybe instead of a color we can think up how to represent web of trust figures now, so we don't chop and change too many times.

I don't plan to implement a deep web-of-trust because it requires loading and storing too much data.  So friends-of-friends is fine, but not friends-of-friends-of-friends.  That means the WoT score will be a simple number:  how many of your friends follow this person.

I guess we could hide the web of trust number in a hover tooltip, and use shades of a color to indicate if the number is low or high.  I dunno.

**Comment by dtonon** [comment]
Coracle show the WoT value near the name (not the picture), but I think it is a information that can be moved to the profile page, or in a popover. I also was thinking about different colors, for example:

- Red: not followed
- Orange: not followed but high WoT value
- Yellow: not followed, but present in a not-mute list

I don't like to use too many colors, but as mentioned the occurrence of profiles with these signs should be minimal, so the impact is equally small.

**Comment by bu5hm4nn** [comment]
As a number nerd, I do want a way to quickly view the WoT score, so I would definitely like to have it in a popover.

> Yellow: not followed, but present in a not-mute list

For me having someone in a list is equivalent to following from a trust perspective.

I do think we should let the user configure aspects of the WoT handling. For me having a person followed by 3-5 people who I follow is a pretty good score (enough to show a "green" indicator if there was one). But that number might be different for other users so I would make it configurable: Maybe `High-Trust Threshold` (Green) and `Medium-Trust Threshold` (Yellow) Everything else is Red.

**Comment by dtonon** [comment]
> For me having someone in a list is equivalent to following from a trust perspective.

Yes, I agree it is not necessary

> I do think we should let the user configure aspects of the WoT handling.

If it's possibile, sure!

> For me having a person followed by 3-5 people who I follow is a pretty good score (enough to show a "green" indicator if there was one). But that number might be different for other users so I would make it configurable: Maybe High-Trust Threshold (Green) and Medium-Trust Threshold (Yellow) Everything else is Red.

I like this.
I'm only no sure about the green flag, maybe I would try to use the accent color for the hight score WoT.

**Comment by bu5hm4nn** [comment]
> I'm only no sure about the green flag, maybe I would try to use the accent color for the hight score WoT.
<img width="1362" alt="Screenshot 2024-06-24 at 15 04 33" src="https://github.com/mikedilger/gossip/assets/3328670/14e2df76-c798-430b-a673-393af84516c1">
😂

**Comment by dtonon** [comment]
Hahaha
Actually Japan has something to say about blu traffic lights!

Jokes aside, since we currently use the accent color for all the basic buttons, and we are not actively inviting the user to act on this profiles, I thought it could be fine to use it without adding a new color to the palette.

**Comment by bu5hm4nn** [comment]
> I thought it could be fine to use it without adding a new color to the palette.

Yes of course
